README.md

```diff
@@ -37,7 +46,12 @@ func main() {
 ## Inline mode
 
 As of January 4, 2016, Telegram added inline mode support for bots.
-Telebot support inline mode in a fancy manner.
+Telebot support inline mode in a fancy manner. Since you are probably
```

Comment:
That's the actual program I worked with. Maybe I should have moved it up to the first example...